src/pages/study-rooms/components/create-study-room-modal.tsx

```diff
@@ -29,8 +30,8 @@ const CreateStudyRoomModal = () => {
 
   const handleClickActionBtn = async () => {
     try {
-      const { id } = await createStudyRoom({ name, intro });
-
+      const { id, participantId } = await createStudyRoom({ name, intro });
+      setLocalStorage({ key: 'participantId', value: String(participantId) });
```

Comment:
이런 인터페이스로 사용되는게 좋을 것 같습니다~
```suggestion
      setLocalStorage('participantId', String(participantId));
```

Reply:
storage Class 개발 후 한번에 수정하려고 합니다.